lib/screens/chat_screen.dart

```diff
@@ -29,11 +31,57 @@ class _ChatScreenState extends State<ChatScreen> {
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      child: SafeArea(
-        //TODO Implement Chat Screen
-        child: Text('Chat screen'),
-      ),
-    );
+    MediaQueryData queryData;
+    queryData = MediaQuery.of(context);
+    return Scaffold(
+        backgroundColor: ChatScreenBackground,
+        appBar: AppBar(
+          backgroundColor: ChatScreenAppBar,
+        ),
+        body: SafeArea(
+            child: Column(
+          mainAxisAlignment: MainAxisAlignment.end,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Padding(
+                padding: const EdgeInsets.all(10.0),
+                child: Row(children: [
+                  Padding(
+                    padding: const EdgeInsets.all(12.0),
+                    child: Align(
```

Comment:
Use `mainAxisAlignment` and `crossAxisAlignment` property to align widgets under `Row` widget